### ISSUE TITLE
Need `diff-pdf` in LaTeX image

### ISSUE THREAD

**Issue by TheChymera** [con/opfvta-replication-2023#30]
Currently our builds are failing, on the immediate level due to a Python data processing error, but in fact because the container environment does not have access to `diff-pdf` to generate the data to being with:

```
root@40c1399916c3:~# make data/paperdiff.log -B
cd prepare; ./paperdiff.sh
./paperdiff.sh: line 15: diff-pdf: command not found
```

I checked and debian [lacks `diff-pdf`](https://repology.org/project/diff-pdf/versions) and only seems [to have `diffpdf`](https://repology.org/project/diffpdf/versions), a similarly named but different and graphical package (I believe we discussed this potential confusion last time we met), which is less well suited to our needs. @yarikoptic am I correct?

We might have to download and build the sources in the image manually, or maybe see if someone just distributes prebuild amd64 executables that work for it. I could try to just copy-paste mine... it's only the one file:

```console
[deco]~ ❱ equery f diff-pdf
 * Searching for diff-pdf ...
 * Contents of app-text/diff-pdf-0.5.1:
/usr
/usr/bin
/usr/bin/diff-pdf
/usr/share
/usr/share/doc
/usr/share/doc/diff-pdf-0.5.1
/usr/share/doc/diff-pdf-0.5.1/AUTHORS
/usr/share/doc/diff-pdf-0.5.1/README.md.bz2
```

Or maybe @yarikoptic wants to make a debian package for it?
Alternatively we can just gentoo everything again, it has the package (and everything else we need for the LaTeX part).
what do you think @asmacdo ?

**Comment by asmacdo** [comment]
i think we should create a diff-pdf container image and use that to avoid dependencies across OSs

**Comment by yarikoptic** [comment]
I googled myself into needing to `apt install pkgconf`

**Comment by TheChymera** [comment]
solved in 2462c048e361f8793a567eff50e8b79efb601d03
